src/Uno.UI.RuntimeTests/Tests/BindingTests/XBindConstControl.xaml.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.UI.Xaml.Controls;
+
+namespace Uno.UI.RuntimeTests.Tests;
+
+public sealed partial class XBindConstControl : Control
+{
+	public const double MyWidth = 200;
```

Comment:
The issue mentions that there is an issue for `private const` which doesn't appear to be tested by this PR. In our binding logic we have `allowPrivateMembers` parameter in some helpers so it's possible there is a code path being passed `false` incorrectly.